### PR TITLE
Release/0.9.0.post6

### PR DESCRIPTION
**Pull request by makseq**
https://github.com/heartexlabs/label-studio/issues/556

### CODE REVIEW

**Comment by makseq** [comment]
Merged https://github.com/heartexlabs/label-studio/pull/531

**Comment by RenaldiMartin** [comment]
Hi

File: label_studio/data_import/templates/import_new.html [hunk 1]
```diff
@@ -36,9 +36,9 @@ <h1>Files uploaded</h1>
         </ul>
       </div>
       <div class="ui checkbox" v-if="show_files_as_tasks_list">
-        <input type="checkbox" id="files_type" @change="toggle_type" ref="files_type" :checked="files_as_tasks_list.selected"/>
+        <input type="checkbox" id="files_type" @change="toggle_type" ref="files_type" :checked="!files_as_tasks_list.selected"/>
         <label for="files_type">
-          Interpret each line in files as a single item
+          <span>Imported files are <b>Time-Series</b> or <b>Text</b></span>
```
Comment (niklub) [inline label_studio/data_import/templates/import_new.html:41]: It has an inverted meaning - tbd also in code if not yet

Comment (makseq) [reply label_studio/data_import/templates/import_new.html:41]: https://github.com/heartexlabs/label-studio/pull/571/files/a27a5e18fe06fe8ac89d9719172cd12a0195317f#diff-18ffdad1c25a1673fc1cc55a73b347dc6cd804b04c6666a91b4a64c86381f6a5R277

**Pull request closed without merging**